themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -79,11 +85,25 @@
   ?>
   <?=$this->render('search/list-' . $viewType . '.phtml', $viewParams)?>
   <?php if (isset($currentSearch['more_link']) && $currentSearch['more_link']): ?>
-    <p>
+    <p class="more_link">
       <a class="icon-link" href="<?=$moreUrl?>">
         <span class="icon-link__label"><?=$this->transEsc($currentSearch['more_link'])?></span>
         <?=$this->icon('more', 'icon-link__icon') ?>
       </a>
     </p>
   <?php endif; ?>
+  <?php if (($currentSearch['more_link'] ?? false) && ($currentSearch['advanced_link'] ?? false)): ?>
+    <p class="advanced_link">
+      <span class="more-advanced-separator"><?=$this->transEsc('combined_more_advanced_separator')?></span>
+  <?php endif; ?>
+  <?php if ($currentSearch['advanced_link'] ?? false): ?>
+    <?php if (!($currentSearch['more_link'] ?? false)): ?>
+      <p>
+    <?php endif; ?>
+      <a class="icon-link" href="<?=$advancedSearchUrl?>">
+        <span class="icon-link__label"><?=$this->transEsc($currentSearch['advanced_link'])?></span>
+        <?=$this->icon('more', 'icon-link__icon') ?>
+      </a>
+   </p>
+  <?php endif; ?>
```

Comment:
The `more_link` is displayed twice, with different text, at the top and bottom of the results.  I haven't looked at the history of that to see if I should do the same here; can do so if helpful.

Reply:
That decision comes out of the original Villanova design, the intent of which was simply to provide multiple access points to the full search results to meet different user needs. The top link is meant to be an "access more options" indicator, whereas the one at the bottom is an "access more results" indicator, but of course they both lead to the same place. This has worked out reasonably well in our own usability testing for more than a decade, but it's not the result of consultation with lots of other institutions. :-)

Reply:
@maccabeelevine, maybe the most future-proof thing to do would be to name the config option in such a way that if we decide to add a second configurable link at the top in the future, it doesn't become confusing... :-)

Reply:
> name the config option in such a way that if we decide to add a second configurable link at the top in the future, it doesn't become confusing

Agreed, and done.